packages/salesforcedx-vscode-apex-replay-debugger/src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,62 @@
+import vscode = require('vscode');
+import TelemetryReporter from 'vscode-extension-telemetry';
+
+export class TelemetryService {
```

Comment:
So this file needs to be duplicated (just like we do for the messages.ts) for each extension that wants to use the telemetry service, right?

Reply:
Yes, this is because we want TelemetryReporter to be initialized with each extension's version and name instead of having that information being a custom field.

Also, core telemetry service has more logic into it that will expose how to show the legal notice message and user setting config to the rest of the extensions while the rest of the extension will have a stripped down version of that.